### PR TITLE
855 article sidebar mobile

### PR DESCRIPTION
**Pull request by Rowansdabomb**

### CODE REVIEW

File: dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js [hunk 1]
```diff
@@ -44,82 +45,117 @@ const tabData = [
   }
 ]
 
-function tabHighlight(localErrors, errors) {
-  for (const error of localErrors) {
-    if (errors.includes(error)){
-      return 'c-article-sidebar__tab-error'
+class ArticleSidebar extends Component {
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js]: For the sake of consistency lets keep these to `...extends React.Component`

Comment (JamieRL) [reply dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js]: Then we can remove the import as well

File: dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js [hunk 2]
```diff
@@ -44,82 +45,117 @@ const tabData = [
...
-            importance={props.article.importance}
-            reading_time={props.article.reading_time}
-            integrations={props.article.integrations}
-            is_breaking={props.article.is_breaking}
-            breaking_timeout={props.article.breaking_timeout}
-            availableIntegrations={props.integrations} />
-        </TabPanel>
+  tabHighlight(localErrors, errors) {
+    for (const error of localErrors) {
+      if (errors.includes(error)){
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js]: space before `{`

File: dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js [hunk 3]
```diff
@@ -44,82 +45,117 @@ const tabData = [
...
+          </TabPanel>
+        </Tabs>
+      </div>
+    )
+  }
+  
+  render () {
+    const isDesktop = window.document.body.clientWidth > desktopSize 
+    const open = isDesktop ? '' : (this.state.isOpen ? 'open': 'closed')
+    const sliderStyle = {
```
Comment (JamieRL) [inline dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js:140]: Would we be able to move this to a style sheet instead?

**Review by JamieRL — CHANGES_REQUESTED** [review]
Just a couple style things. Looks good. Yayyy mobile compatibility!

File: dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js (hunk 1 above)
Comment (Rowansdabomb) [reply dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js]: good idea

File: dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js (hunk 3 above)
Comment (Rowansdabomb) [reply dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js:140]: woops

**Review by JamieRL — APPROVED** [review]

**Pull request merged**